index.css

```diff
@@ -0,0 +1,377 @@
+body {
+    font-family: GT Eesti Pro Display, arial, sans-serif;
+}
+@font-face {
+    font-family: GT Eesti Pro Display, arial, sans-serif;
+}
+.background{
+    background-color : #01090B
+}
+
+a, input, img {
+    outline: none;
+    text-decoration: none;
+    display: block;
+}
+.item-a {
+    grid-area: header;
+}
+
+.item-b {
+    grid-area: main;
+}
+.item-c {
+    grid-area: footer;
+}
+
+
+.container {
+    background: url("static/1+1.jpg");
+    margin: -8px;
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: auto;
+    grid-template-areas:
+    "header"
+    "main"
+    "footer";
+    background-size: cover;
+    height: 640px;   // костыль
+}
+
+
+
+.fontGT{
+    font-family: "GT-Eesti-Pro-Display-Bold",cursive;
+    font-style: normal;
+    font-weight: bold;
+    color: #FFFFFF;
+}
+
+.navbar {
+
+    display: grid;
+    grid-template-columns: 150px 150px 150px 150px 150px auto 50px auto 100px;
+    column-gap: 10px;
+    font-size: 24px;
+    line-height: 28px;
+    align-items: flex-end;
+    text-align: center;
+    background: linear-gradient(180deg, #070606 6.08%, rgba(7, 6, 6, 0.5) 55.03%, rgba(14, 12, 12, 0.25) 78.99%, rgba(27, 23, 23, 0.1) 90.45%, rgba(27, 23, 23, 0) 100%);
+}
+
+input[type=text] {
+    width: 130px;
+    -webkit-transition: width 0.4s ease-in-out;
+    transition: width 0.4s ease-in-out;
+}
+
+/* When the input field gets focus, change its width to 100% */
+input[type=text]:focus {
+    width: 100%;
+}
+
+
+.label{
+    place-self: center;
+}
+
+.mainMovie {
+    display: grid;
+    grid-template-columns: auto  600px 30px ;
+    column-gap: 10px;
+    grid-template-rows: 200px auto 20px;
+    row-gap: 10px;
+    grid-template-areas:
```

Comment:
https://blog.logrocket.com/flexbox-vs-css-grid/#:~:text=CSS%20grid%20is%20better%20when%3A&text=We%20can%20define%20the%20gap,re%20working%20with%20many%20breakpoints